lcm/lcm_trajectory.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include <Eigen/Dense>
+#include <vector>
+#include <string>
+#include <unordered_map>
+#include "drake/systems/lcm/serializer.h"
+
+#include "dairlib/lcmt_saved_traj.hpp"
+
+namespace dairlib {
+
+class LcmTrajectory {
+ public:
+  struct Trajectory {
+    Trajectory() {}
+    Trajectory(std::string traj_name, const lcmt_trajectory_block& traj_block);
+
+    std::string traj_name;
+    Eigen::VectorXd time_vector;
+    Eigen::MatrixXd datapoints;
+    std::vector<std::string> datatypes;
+  };
+
+  LcmTrajectory() {}
+  LcmTrajectory(const std::vector<Trajectory>& trajectories,
+                const std::vector<std::string>& trajectory_names,
+                const std::string& name = "DEFAULT_NAME",
+                const std::string& description = "DEFAULT_DESCRIPTION");
+
+  explicit LcmTrajectory(const lcmt_saved_traj& traj);
+
+  // Writes this LcmTrajectory object to a file specified by filepath
```

Comment:
Document what happens in the case of failure (if the file cannot be created)

Reply:
The header for these functions should indicate that an exception will be thrown in case of an error.